docs/development/running-tests/README.md

```diff
@@ -701,15 +701,18 @@ you can access both from inside a VM with nat/bridged networking as follows:
 
 ```shell
 # Start ng serve middleware binding to all interfaces
-PROXY_HOSTNAME=<your local IP address> npm run serve
+FE_HOST=<your local IP address> PROXY_HOSTNAME=<your local IP address> npm run serve
 
 # Start your openproject server with the CLI proxy configuration set
-OPENPROJECT_CLI_PROXY='http://<your local ip>:4200' ./bin/rails s -b 0.0.0.0 -p 3000
+OPENPROJECT_DEV_EXTRA_HOSTS=<your local IP address> OPENPROJECT_HOST_NAME=<your local IP address> OPENPROJECT_CLI_PROXY='http://<your local ip>:4200' ./bin/rails s -b 0.0.0.0 -p 3000
```

Comment:
Additional comments, not sure how helpful they are for this context

Instead of passing everything on the command line, I adjusted my .env instead with the following

```.env
LOCAL_IP_ADDR='192.168.x.y'
OPENPROJECT_DEV_EXTRA_HOSTS=$LOCAL_IP_ADDR
OPENPROJECT_HTTPS=false
OPENPROJECT_HOST_NAME=$LOCAL_IP_ADDR
OPENPROJECT_CLI_PROXY="http://$LOCAL_IP_ADDR:4200"
```

Also it's important that OPENPROJECT_HTTPS is `false` (I usually have it set to true on my local env), otherwise I was not able to authenticate into the project.

Reply:
Cheers, I think the idea would be to keep it one-time run commands- easily copy paste & run. Setting the env vars on a file is the a user decision 🤷🏾 

Mentioning `OPENPROJECT_HTTPS ` might be overkill as by default most people are not on docker TLS, and have it off. If we mention it, it then follows that we have to instruct people using docker on how to set external device access 😟